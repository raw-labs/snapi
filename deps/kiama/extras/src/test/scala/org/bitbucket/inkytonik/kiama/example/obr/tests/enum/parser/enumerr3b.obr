(* Missing comma between enumeration constant identifiers (not first) *)

PROGRAM Err803b (i : INTEGER) : INTEGER;

VAR
    x : ( harold, bob eugene );

BEGIN
END Err803b.
