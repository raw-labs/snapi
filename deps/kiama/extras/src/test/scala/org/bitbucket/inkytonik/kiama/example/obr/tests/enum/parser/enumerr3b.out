src/test/scala/org/bitbucket/inkytonik/kiama/example/obr/tests/enum/parser/enumerr3b.obr:6:23:error: ')' expected but 'e' found
    x : ( harold, bob eugene );
                      ^
